kubernetes/istio_details_service.go

```diff
@@ -48,9 +48,30 @@ func (in *IstioClient) GetIstioDetails(namespace string, serviceName string) (*I
 	return &istioDetails, nil
 }
 
+// CreateIstioObject creates an Istio object
+func (in *IstioClient) CreateIstioObject(api, namespace, resourceType, name string, json string) (IstioObject, error) {
+	var result runtime.Object
+	var err error
+
+	if api == configGroupVersion.Group {
```

Comment:
I think we need to review this, in a previous PR it has been introduced another group, so this needs another comparison case.

Reply:
https://github.com/kiali/kiali/pull/793/files#diff-b6924f0a528196fa3f33089981235463R314